syntaxes/samarium.tmLanguage.json

```
@@ -120,7 +120,7 @@
                 },
                 {
                     "name": "keyword.control.samarium",
-                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%)"
+                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%<=)"
```

You missed the pipe here
```suggestion
                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=)"
```

---

thanks for input